app/models/passenger.rb

```diff
@@ -24,12 +25,29 @@ class Passenger < ApplicationRecord
 
   belongs_to :mobility_device, optional: true
 
+  before_validation do
+    if active_status_changed? && active?
+      assign_attributes(registration_date: Time.zone.today)
+    elsif registration_date.blank?
+      assign_attributes(registration_date: (created_at || Time.zone.today))
+    end
+  end
+
   def expiration_display
     return if permanent?
 
     doctors_note.try(:expiration_date).try :strftime, '%m/%d/%Y' || 'No Note'
   end
 
+  def rides_expire
+    return if permanent?
+
+    return doctors_note.expiration_date if doctors_note.present?
+    return registration_date + 3.days if persisted?
+
+    3.days.since.to_date
```

Comment:
I like using the method `since` much more than the plus sign, personally.

Reply:
Personally, I've never been able to parse how the word "since" on its own means a date or time in the future. Luckily, Rails agrees with me and has an alias:

```ruby
4.days.from_now
```

`.since` reads much better with the optional argument:

```ruby
4.days.since(my_birthday)
4.days.after(my_birthday)
```

Reply:
I will choose `from_now` to resolve this point of contention since I agree that `since` isn't a great method name (although still better than a plus)